src/common/api/saving/recommendations.js

```diff
@@ -6,12 +6,12 @@ import { getCurrentLanguageCode } from '../../helpers'
 export function getRecommendations(resolved_id) {
   var lang = getCurrentLanguageCode()
   return request({
-    path: 'getSuggestedItems/',
+    path: 'v3/discover/recIt',
```

Comment:
Not sure whether to include `v3/` here or not.

Reply:
No need to add it. Our request function appends it. This will work as `discover/recIt`

Reply:
No need to include `v3/`. Our request function prepends it. You can get away with just `discover/recIt`